shapely/tests/geometry/test_collection.py

```diff
@@ -82,6 +82,32 @@ def test_geointerface(geometrycollection_geojson):
     assert geom.__geo_interface__ == geometrycollection_geojson
 
 
+def test_mapping():
+    geom = GeometryCollection([
+        Polygon(((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0)))
+    ])
+    # force_ccw=False
+    assert mapping(geom, force_ccw=False) == {
+        "type": "GeometryCollection",
+        "geometries": [{
+            "type": "Polygon",
+            "coordinates": (
+                ((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0), (0.0, 0.0)),
+            )
+        }]
+    }
+    # force_ccw=True
```

Comment:
This should be a separate test function, so that failure in the first case doesn't cover up failure or success of the second case.